common/protob/messages-common.proto

```diff
@@ -107,31 +107,15 @@ message PinMatrixAck {
  * @next PassphraseAck
  */
 message PassphraseRequest {
-    optional bool on_device = 1;    // passphrase is being entered on the device
```

Comment:
let's keep this field in the protobuf and rename it to something like `_deprecated_on_device`, so that a newer trezorlib can continue to work with older TT